ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/chain/DefaultBlockchainTest.java

```diff
@@ -734,11 +793,19 @@ private void assertTotalDifficultiesAreConsistent(final Blockchain blockchain, f
     assertEquals(td, blockchain.getChainHead().getTotalDifficulty());
   }
 
-  private DefaultMutableBlockchain createBlockchain(
+  private BlockchainStorage createStorage(final KeyValueStorage kvStore) {
+    return new KeyValueStoragePrefixedKeyBlockchainStorage(
+        kvStore, new MainnetBlockHeaderFunctions());
+  }
+
+  private DefaultBlockchain createMutableBlockchain(
```

Comment:
Does this need to be `DefaultBlockchain` or can it simply be `Blockchain`? i.e. do the tests need methods on `DefaultBlockchain` or is the type needed for other calls?

Reply:
Some of the tests access protected methods, so just cast it here for simplicity.